### PR TITLE
Feature/calculate area and perimeter for few figures

### PR DESCRIPTION
**Pull request by ChebanTetyana**
Task about creating 4 figures, one parent. Create a Calculator that passes any of them to one function, which calculates the area and perimeter using the interface. Write unit test units for them. Use double test (with stub).

### CODE REVIEW

File: Tests/Models/TriangleTest.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Cheba\PhpUnit\Tests\Models;
+
+use Cheba\PhpUnit\Models\Triangle;
+use PHPUnit\Framework\TestCase;
+
+class TriangleTest extends TestCase
+{
+    public function testArea()
```
Comment (timiTao) [inline Tests/Models/TriangleTest.php]: define for all test retur type `:void`

File: Tests/Services/ShapeCalculatorTest.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+class ShapeCalculatorTest extends TestCase
+{
+    public function testCalculateArea()
+    {
+        $calculator = new ShapeCalculator();
+
+        $rectangle = new Rectangle(4, 5);
+        $this->assertEquals(20, $calculator->calculate($rectangle));
+
+        $circle = new Circle(3);
```
Comment (timiTao) [inline Tests/Services/ShapeCalculatorTest.php]: that is non needed as you depends on interface

File: Tests/Models/TriangleTest.php (hunk 1 above)
Comment (ChebanTetyana) [reply Tests/Models/TriangleTest.php]: Added return type for all methods.

File: Tests/Services/ShapeCalculatorTest.php (hunk 2 above)
Comment (ChebanTetyana) [reply Tests/Services/ShapeCalculatorTest.php]: Removed class ShapeCalculatorTest.

**Comment by timiTao** [comment]
That is ok, resolve conflicts and merge

**Pull request merged**